packages/validator/src/services/syncCommitteeDuties.ts

```diff
@@ -15,7 +15,7 @@ import {ValidatorStore} from "./validatorStore";
 /** Only retain `HISTORICAL_DUTIES_PERIODS` duties prior to the current periods. */
 const HISTORICAL_DUTIES_PERIODS = 2;
 /** Epochs prior to `ALTAIR_FORK_EPOCH` to start fetching duties */
-const ALTAIR_FORK_LOOKAHEAD_EPOCHS = 1;
+const ALTAIR_FORK_LOOKAHEAD_EPOCHS = 0;
```

Comment:
Add a nice chunk of text in the JSDocs of this variable with context and linking to the issue that started this PR